i18n/locales/uk-UA/main.ftl

```diff
@@ -0,0 +1,72 @@
+site-description = { $n-icons } Безкоштовні значки { $svg } популярних брендів
+close = Закрити
+theme = Тема
+light-color-scheme = Світла тема
+dark-color-scheme = Темна тема
+system-color-scheme = Системна
+layout = Розкладка
+comfortable = Зручна
+compact = Компактна
+open-search-bar = Панель пошуку
+open-controls = Панель інструментів
+order = Впорядковувати
+sort-alphabetically = За алфавітом
+sort-by-color = За кольором
+sort-by-search-match = За відповідністю
+svg = SVG
+colored-svg = кольоровий SVG
+pdf = PDF
+download = Завантажити
+download-filetype = Завантажити { $filetype }
+search = Пошук
+search-by-brand = Шукати бренд…
+clear-search = Очистити
+maintained-by = Проєкт є суспільним надбанням { $license } та підтримується { $maintainers }.
+cco = CCO
+github = GitHub
+open-collective = Open Collective
+simple-icons-contributors = учасниками Simple Icons
+use-platform = Використовуйте { $platform } для надсилання запитів на покращення та участі.
+supported-by = Підтримується за рахунок ваших донатів через { $platform }.
+made-on = Зроблено з ❤️ на GitHub
+icon-missing = Немає значка?
+submit-a-request = Зробіть запит
+icon-outdated = Значок застарів?
+report-outdated-icon = Повідомте про застарілий значок
+share-this = Поділитись в Х!
+load-more-icons = Завантажити ще
+go-to-header = Вгору
+go-to-footer = Вниз
+deprecated = Застаріле
+will-be-removed-at = { $icon } буде вилучено у версії v{ $version }
+will-be-removed-at-extended = { $icon } буде вилучено { $date } у версії { $version } (див { $pr })
+scalable-vector-graphic = Scalable Vector Graphic
+third-party-extensions = Сторонні розширення
+main-repository = Основний репозиторій
+jsdelivr = jsDelivr (Content Delivery Network)
+unpkg = UNPKG (Content Delivery Network)
+legal-disclaimer = Юридичні застереження
+open-menu = Відкрити меню
+close-menu = Закрити меню
+change-language = Змінити мову
+select-a-language = Оберіть мову
+copy-icon-slug = Копіювати { $icon } мітку ({ $slug })
+brand-guidelines = Настанови бренду
+license = Ліцензія
+custom-license = Власна ліцензія
+subject-icon = занчок { $icon }
+copy-icon-svg = Копіювати { $icon } SVG
+error-generating-pdf = Помилка створення PDF-файлу за допомогою бібліотеки PDFKit:
+view-icon = Переглянути { $icon }
+copy-hex-color = Копіювати hex колір
+discord = Discord
+brand = Бренд
+color = Колір
+path = Шлях
+upload-svg = Надіслати SVG
+save-preview = Зберегти попередній перегляд
+icons = Занчки
+preview-generator = Генератор попереднього перегляду
+page-not-found = Сторінку не знайдено
+maybe-youre-looking-for = Можливо, ви шукаєте…
+copy-preview = Копіювати попередній перегляд
```

Comment:
I'm seeing a problem with styles in the [`/preview`](https://wasm.simpleicons.org/preview) endpoint.

![image](https://github.com/mondeja/simple-icons-website-rs/assets/23049315/f63c239c-2c8f-4d01-81d3-2334083a5314)

It would be possible to remove the word `попередній` from `save-preview` and `copy-preview` translations?

```suggestion
save-preview = Зберегти перегляд
icons = Занчки
preview-generator = Генератор попереднього перегляду
page-not-found = Сторінку не знайдено
maybe-youre-looking-for = Можливо, ви шукаєте…
copy-preview = Копіювати перегляд
```

![image](https://github.com/mondeja/simple-icons-website-rs/assets/23049315/ba48aef7-8126-4ebe-8103-a5ffbaf47a2a)

I've done the same with the Japanese translation. It would be great to maintain the same style as the word doesn't seem to add too much meaning to the buttons, but I'm not sure as I don't speak Ukrainian.

Reply:
I may suggest replacing `попередній перегляд` with direct transliteration of word `preview` -> `превью` in both cases.

Reply:
Yeap, that would be great.